packages/nodes-base/nodes/Code/SandboxPython.ts

```diff
@@ -0,0 +1,264 @@
+import { normalizeItems } from 'n8n-core';
+import { ValidationError } from './ValidationError';
+import { ExecutionError } from './ExecutionError';
+import { CodeNodeMode, isObject, REQUIRED_N8N_ITEM_KEYS } from './utils';
+import type { python, py } from 'pythonia';
```

Comment:
Instead of `pythonia` can we please use one of the python interpreters instead?
These are some of the options:
1. https://brython.info/index.html
2. https://pyodide.org/en/stable/
3. https://github.com/jspython-dev/jspython
Adding another out-of-process runtime like python would require us to sandbox these runtimes. most interpreters however are sandboxed by default.

Reply:
Replaced it with pyodide